go/plek.go

```diff
@@ -44,8 +44,8 @@ var httpDomains = map[string]bool{
 // corresponding PLEK_SERVICE_FOO_URI environment variable.  For example, to
 // override the "foo-api" service url, set PLEK_SERVICE_FOO_API_URI to the base
```

Comment:
Should we update `service url` here in the comment as well? Since it's not really a URL.